### DISCUSSION TITLE
In cli, why no option named "--output-file-name" ?

### DISCUSSION THREAD

**Discussion by SprayHank** [webpack/webpack#18387] [Q&A]
## Feature request
add the option named "--output-file-name", and I think the function is very clear from the option name.

**What is the expected behavior?**
Can specify the output file name without a configuration file
**What is motivation or use case for adding/changing the behavior?**
Can specify the output file name without a configuration file
**How should this be implemented in your opinion?**
I do not read the source code yet, so I do not know now.
**Are you willing to work on this yourself?**
yes

**Comment by alexander-akait** [comment]
Please run `webpack --help`, we have `webpack --output-filename ./my-file.js`
